### DISCUSSION TITLE
Does Antd support Rspack bundler?

### DISCUSSION THREAD

**Discussion by AsuraKev** [ant-design/ant-design#44873] [Q&A] [answered]
Currently Antd supports Webpack and Vite and Next, is it possible to support Rspack in the future?

**Comment by yoyo837** [comment, marked as answer]
We haven't tried it yet. `rspack` claims to be fully compatible with the `webpack` ecosystem. You can have a try. If you encounter problems, you can provide feedback directly to `rspack`.
